frontend/projects/valtimo/case/src/lib/models/case-widget.model.ts

```diff
@@ -159,6 +160,21 @@ interface CustomCaseWidgetConfig {
   [componentKey: string]: Type<any>;
 }
 
+enum Operator {
+  NOT_EQUAL_TO = '!=',
+  EQUAL_TO = '==',
```

Comment:
Can these not be reused from dashboard? If not, move it to shared lib so it's only defined and used once

Reply:
I moved these to shared and use them in both dashboard and widgets. Also moved all translations to the general translation files so they are re-used everywhere.